client/core/exchangeratefetcher.go

```diff
@@ -32,11 +33,33 @@ const (
 )
 
 var (
-	dcrDataURL     = "https://explorer.dcrdata.org/api/exchangerate"
+	dcrDataURL = "https://explorer.dcrdata.org/api/exchangerate"
+	// coinpaprika has two options. /tickers is for the top 2500 assets all in
+	// one request. /ticker/[slug] is for a single ticker. From testing
+	//    Single ticker request took 274.626125ms
+	//    Size of single ticker response: 0.733 kB
+	//    All tickers request took 47.651851ms
+	//    Size of all tickers response: 1828.863 kB
+	// Single ticker requests were < 1 kB, while all tickers were 1.8 MB, but
+	// the larger request was faster (without json decoding). Coinpaprika's,
+	// free tier allows up to 25k requests per month. For a
+	// fiatRateRequestInterval of 12 minutes, that 3600 requests per
+	// month for the all tickers request, and 3600 * N requests for N assets.
+	// So any more than 25000 / 3600 = 6.9 assets, and we can expect to run into
+	// rate limits. But the bandwidth of the full tickers request is kinda
+	// ridiculous too. Solution needed.
```

Comment:
Could keep track of how many requests are made, and if it gets to point where it can only make it through the rest of the month if only the all assets request is used, then switch to the all assets request. For most people it won't make it to that point because they don't keep dexc on all the time.